types/index.ts

```diff
@@ -57,3 +57,8 @@ export interface INotes {
   timestamp: number;
   created_at: string;
 }
+
+export interface ISliceState {
+  notesArr: INotes[];
```

Comment:
`INotes` is a confusing variable name tbh. 
I can see its a single `Note` object. So it would be better (and potentially less confusing going forward if you name it `INote`) instead. 

so final result would be 

`notesArr : INote[]`

Reply:
Makes sense. Renamed it.